### PR TITLE
APIS-4876 Rebranded to ip-allowlisting

### PR DESCRIPTION
**Pull request by adampridmore**

### CODE REVIEW

File: app/model/Forms.scala [hunk 1]
```diff
@@ -194,7 +194,7 @@ object Forms {
     def fromSetOfWhitelistedIps(whiteListedIps: Set[String]) = Some(whiteListedIps.mkString("\n"))
 
     val form: Form[Set[String]] = Form(mapping(
-      "whitelistedIps" -> text.verifying(whitelistedIpsConstraint)
+      "allowlistedIps" -> text.verifying(whitelistedIpsConstraint)
```
Comment (AndySpaven) [inline app/model/Forms.scala:197]: change name of val too ?

File: app/model/Forms.scala [hunk 2]
```diff
@@ -194,7 +194,7 @@ object Forms {
     def fromSetOfWhitelistedIps(whiteListedIps: Set[String]) = Some(whiteListedIps.mkString("\n"))
```
Comment (AndySpaven) [inline app/model/Forms.scala:194]: Name of function and val ?

**Review by AndySpaven — COMMENTED** [review]
Probably should change everything we can even if it's not front facing.  I think application.whitelistedIPs would need to remain unchanged...

**Review by AndySpaven — APPROVED** [review]

File: app/model/Forms.scala (hunk 1 above)
Comment (adampridmore) [reply app/model/Forms.scala:197]: Discussed with @AndySpaven : We're only going to change references that are returned to the browser. We're not going to fully refactor the code to use the new terminology.

**Pull request merged**